platform/jvm/capture/src/test/kotlin/io/bitdrift/capture/fakes/FakeDateProvider.kt

```diff
@@ -0,0 +1,21 @@
+// capture-sdk - bitdrift's client SDK
+// Copyright Bitdrift, Inc. All rights reserved.
+//
+// Use of this source code is governed by a source available license that can be found in the
+// LICENSE file or at:
+// https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt
+
+package io.bitdrift.capture.fakes
+
+import io.bitdrift.capture.providers.DateProvider
+import java.util.Date
+
+/**
+ * Fake [DateProvider] for use in tests.
+ * Returns a fixed timestamp by default, but can be configured to return different values.
```

Comment:
/nit: but wouldn't the best way to configure this stub is to receive the value you want it to return in the ctor?